### ISSUE TITLE
Failed to load expected config..............erl_parse, "bad term"

### ISSUE THREAD

**Issue by shlokspeare** [msgpack-rpc/msgpack-rpc-erlang#2]
Unable to use msgpack-rpc-erlang as a dependency in my code. Keep getting the above error.

1) Have installed the requisite deps - including ranch (branch master), proper, & msgpack (branch master).

2) Tried to make changes in rebar.config and add but get the error in subject line
{deps, [
{msgpack-rpc, ".*", {git, "git://github.com/msgpack-rpc/msgpack-rpc-erlang.git", "master"}},......

**Comment by kuenishi** [comment]
You should write as `{msgpack_rpc, ".*", {git, "git://github.com/msgpack-rpc/msgpack-rpc-erlang.git", "master"}}` in your rebar.config.

**Comment by shlokspeare** [comment]
Yes, plus even rename the folder as msgpack_rpc instead of msgpack-rpc in deps sub-directory, othewise .app file is not found-able. Thank you
